engines/cge2/cge2.h

```diff
@@ -157,10 +157,13 @@ class CGE2Engine : public Engine {
 	bool hasFeature(EngineFeature f) const override;
 	bool canSaveGameStateCurrently() override;
 	bool canLoadGameStateCurrently() override;
+	Common::Language getLanguage() const;
 	Common::Error saveGameState(int slot, const Common::String &desc, bool isAutosave = false) override;
 	Common::Error loadGameState(int slot) override;
 	Common::Error run() override;
 
+	Common::TextToSpeechManager *_ttsMan;
```

Comment:
One more comment: I don't think you need to make this a class member variable. It seems to be used in a single place (in `CGE2Engine::run()` to set the language), so it would be simpler to just call `g_system->getTextToSpeechManager()` there rather than store it as a class variable.

Reply:
ok, moved.